js/id/renderer/mapillary_sign_layer.js

```diff
@@ -9,9 +9,20 @@ iD.MapillarySignLayer = function(context) {
         currentImage,
         svg, image_preview_div, request, signs_defs;
 
-    request = d3.json('css/traffico-release-0.1.5/global-patched.json', function(error, data) {
+    var signs_defs={};
```

Comment:
lint error `12:9   error  signs_defs is already defined             no-redeclare`